src/vlrdevapi/_client.py

```
@@ -81,11 +91,30 @@ def __init__(
+        self.team = TeamNamespace(self._client, self.timeout, self.retry_config, self._rate_limiter, merged_headers, self._source_tz)
+        self.event = EventNamespace(self._client, self.timeout, self.retry_config, self._rate_limiter, merged_headers, self._source_tz)
+
+    def _detect_timezone(self) -> ZoneInfo | tzinfo | None:
+        """Detect the viewer timezone VLR.gg renders for this client session."""
+        try:
+            from selectolax.parser import HTMLParser
+
+            response = self._client.get(REFERENCE_MATCH_PATH, timeout=self.timeout)
+            response.raise_for_status()
+            return detect_vlr_timezone(HTMLParser(response.text))
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)